### ISSUE TITLE
k8s.io/docs/setup/independent/install-kubeadm: docker installation step is wrong

### ISSUE THREAD

**Issue by flx42** [kubernetes/website#8554]
```
add-apt-repository "deb https://download.docker.com/linux/$(. /etc/os-release; echo "") $(lsb_release -cs) stable"
```

This `$(. /etc/os-release; echo "")` is not useful, it won't work since it's supposed to be `https://download.docker.com/linux/ubuntu` or `https://download.docker.com/linux/debian`

I think you meant `$(. /etc/os-release; echo "$ID")`

**Comment by ajaeger** [comment]
Fixed with #8527

**Comment by neolit123** [comment]
> Fixed with #8527

+1

/close
